### PR TITLE
Do not build libtensorflow-jni.so on android

### PR DESCRIPTION
**Pull request by chunseoklee**
set BUILD_ANDROID_TFLITE and BUILD_ANDROID_BENCHMARK_APP OFF

ONE-DCO-1.0-Signed-off-by: Chunseok Lee <chunseok.lee@samsung.com>

### CODE REVIEW

File: infra/nnfw/cmake/options/options_aarch64-android.cmake [hunk 1]
```diff
@@ -3,8 +3,8 @@
 # NOTE BUILD_ANDROID_TFLITE(JNI lib) is disabled due to BuiltinOpResolver issue.
 # tensorflow-lite does not build BuiltinOpResolver but JNI lib need it
 # Related Issue : #1403
-option(BUILD_ANDROID_TFLITE "Enable android support for TensorFlow Lite" ON)
-option(BUILD_ANDROID_BENCHMARK_APP "Enable Android Benchmark App" ON)
+option(BUILD_ANDROID_TFLITE "Enable android support for TensorFlow Lite" OFF)
+option(BUILD_ANDROID_BENCHMARK_APP "Enable Android Benchmark App" OFF)
```
Comment (hseok-oh) [inline infra/nnfw/cmake/options/options_aarch64-android.cmake:7]: @chunseoklee Is there any reason to disable `BUILD_ANDROID_BENCHMARK_APP` ?

**Comment by hseok-oh** [comment]
Need to update `runtime/contrib/android/api/Prebuilt.mk`

File: infra/nnfw/cmake/options/options_aarch64-android.cmake (hunk 1 above)
Comment (chunseoklee) [reply infra/nnfw/cmake/options/options_aarch64-android.cmake]: @hseok-oh It was a mistake. updated.

**Comment by chunseoklee** [comment]
> Need to update `runtime/contrib/android/api/Prebuilt.mk`

I will update after local aar build test.

**Comment by chunseoklee** [comment]
@hseok-oh I have locally built android aar with this PR

**Review by hseok-oh — APPROVED** [review]
LGTM

**Review by wateret — APPROVED** [review]
LGTM

**Pull request merged**